### PR TITLE
Add aws_template_storage_region_name and improve aws_template descrip…

### PR DESCRIPTION
**Pull request by ghost**
…tions. Link to AWS advanced topic.

## Description
https://jira.mesosphere.com/browse/DOCS-2017

## Urgency
- [ ] Blocker 
- [ ] High
- [ x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.10/installing/custom/configuration/configuration-parameters.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ This topic provides all available configuration parameters. Except where explici
...
-| [aws_template_storage_bucket_path](#aws_template_storage_bucket_path)  | The S3 bucket storage path. |
-| [aws_template_upload](#aws_template_upload)                            | Indicates whether to automatically upload the customized advanced templates to your S3 bucket. |
-| [aws_template_storage_access_key_id](#aws_template_storage_access_key_id)         | The AWS [access key ID](http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys). |
-| [aws_template_storage_secret_access_key](#aws_template_storage_secret_access_key) | The AWS [secret access key](http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys). |
+| aws_template_storage_access_key_id         | The [access key ID](http://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) of the account owning the AWS S3 bucket. |
+| aws_template_storage_bucket                | The name of an S3 bucket to contain [customized advanced AWS templates](/1.10/installing/cloud/aws/advanced/#create-your-templates). |
+| aws_template_storage_bucket_path           | The path to the S3 bucket. |
```
Comment (branden) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: This is a little pedantic, but this path points to a storage location _within_ the S3 bucket. The other `aws_template_storage_*` params get us to the bucket, and then we use this param to decide where in the bucket's filesystem to store deployment artifacts.

**Review by branden — APPROVED** [review]
LGTM, one small suggestion below.

File: 1.10/installing/custom/configuration/configuration-parameters.md (hunk 1 above)
Comment (stbof) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: Not at all pedantic. I've corrected it to say:

The path to a location within the S3 bucket to store template artifacts.

Comment (branden) [reply 1.10/installing/custom/configuration/configuration-parameters.md]: Looks great 👍

**Pull request merged**